task1.py

```diff
@@ -0,0 +1,6 @@
+print("Multi-paradigm programming language:Task 1")
+print("Maslova Vitaliia, i dont know")
+s = ("Vladislav Valeriyovich Ovsyannikov, " * 45)
+a = len(s)
+print(s[:a - 2])
```

Comment:
in python you can apply negative indexes for slicing.
for ex. -1 is the last item. -2 the item before the last one and etc,
so, you don't need to calculate len() - 2, can just leave -2